rna/rna-quantification/rna-quantification.inputs.json

```diff
@@ -0,0 +1,20 @@
+{
+  "##_COMMENT1": "Set your per-sample values: name and fastq files",
+  "RNAQuantification.sample_name": "${this.sample_id}",
+  "RNAQuantification.fastq_1": "${this.fastq2}",
+  "RNAQuantification.fastq_2": "${this.fastq1}",
+
+  "##_COMMENT2": "Path to the reference GTF file and salmon index file",
+  "RNAQuantification.gene_map": "gs://amp-pd-transcriptomics/inputs/reference/gencode/gencode.v29.primary_assembly.annotation.gtf.gz",
+  "RNAQuantification.salmon_index": "gs://amp-pd-data-rna-workflows/inputs/reference/salmon/Homo_sapiens.gencode.v29.all.salmon0.11.3_gencodeOption.index.tar.gz",
+
+  "##_COMMENT3": "Path to a Docker image containing salmon",
+  "RNAQuantification.salmon_docker": "combinelab/salmon:0.11.3",
+  
+  "RNAQuantification.runtime_zones": "us-central1-c us-central1-b",
```

Comment:
Let's be consistent with the zones we include as examples.
Let's standardize on `us-central1-a us-central1-b` in the inputs.json and the README.md.